<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>616</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1187</width>
    <height>616</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1187</width>
    <height>616</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Успеваемость</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1171</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Просмотр оценок для курсантов</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1151</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>Журнал оценок</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>1131</width>
         <height>461</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <column>
        <property name="text">
         <string>ФИО</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Номер группы</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ФП</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ЦОС</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ТЭС</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>МП</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ОС</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Итог</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Просмотр журнала для КОФ</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1151</width>
        <height>321</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>ФИО</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер группы</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ФП</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ЦОС</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ТЭС</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>МП</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ОС</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Итог</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>561</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Итог:</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество отличнкиов:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество хорошистов:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество троичников:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Количество двоичников:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Процент отличнкиов:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Процент хорошистов:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Процент троичников:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>110</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Процент двоичников:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>110</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>340</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Вывести отчет в Exsel:</string>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Вывести</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Добавить запись для журналиста</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>821</width>
        <height>461</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>ФИО</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер группы</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ФП</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ЦОС</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ТЭС</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>МП</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ОС</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Итог</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>321</width>
        <height>301</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить запись:</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ФИО:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>251</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Петров П.П.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите оценку за ФП:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>100</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>260</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите оценку за ЦОС:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите оценку заТЭС:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите оценку за МП:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>220</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Введите оценку за ОС:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Номер группы:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>314</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>1151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Подключиться к серверу:</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>8686</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>31</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IP-адрес:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>31</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_18">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>221</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>Создать базу данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Закрыть приложение</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Б</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
